`bevy` fails to build with just `bevy_dev_tools` feature

## Bevy version

Main branch :)

## \[Optional\] Relevant system information

Caught on both the `ubuntu-latest` Github Actions runner and on MacOS.

## What you did

```shell
cargo c -p bevy --no-default-features -F bevy_dev_tools
```

## What went wrong

```bash
error[E0609]: no field `render_layers` on type `&mut GizmoConfig`
   --> crates/bevy_dev_tools/src/ui_debug_overlay/mod.rs:112:20
    |
112 |             config.render_layers = LAYOUT_DEBUG_LAYERS.clone();
    |                    ^^^^^^^^^^^^^ unknown field
    |
    = note: available fields are: `enabled`, `line_width`, `line_perspective`, `line_style`, `depth_bias`, `line_joints`

For more information about this error, try `rustc --explain E0609`.
error: could not compile `bevy_dev_tools` (lib) due to 1 previous error
```

## Additional information

Originally caught in [this run](https://github.com/TheBevyFlock/flag-frenzy/actions/runs/10335648662/job/28610474579) by [`flag-frenzy`](https://github.com/TheBevyFlock/flag-frenzy).

--- a/crates/bevy_dev_tools/Cargo.toml
+++ b/crates/bevy_dev_tools/Cargo.toml
@@ -22,7 +22,9 @@ bevy_core = { path = "../bevy_core", version = "0.15.0-dev" }
 bevy_core_pipeline = { path = "../bevy_core_pipeline", version = "0.15.0-dev" }
 bevy_diagnostic = { path = "../bevy_diagnostic", version = "0.15.0-dev" }
 bevy_ecs = { path = "../bevy_ecs", version = "0.15.0-dev" }
-bevy_gizmos = { path = "../bevy_gizmos", version = "0.15.0-dev" }
+bevy_gizmos = { path = "../bevy_gizmos", version = "0.15.0-dev", features = [
+  "bevy_render"
+] }
 bevy_hierarchy = { path = "../bevy_hierarchy", version = "0.15.0-dev" }
 bevy_input = { path = "../bevy_input", version = "0.15.0-dev" }
 bevy_math = { path = "../bevy_math", version = "0.15.0-dev" }

--- a/crates/bevy_dev_tools/Cargo.toml
+++ b/crates/bevy_dev_tools/Cargo.toml
@@ -23,7 +23,7 @@ bevy_core_pipeline = { path = "../bevy_core_pipeline", version = "0.15.0-dev" }
 bevy_diagnostic = { path = "../bevy_diagnostic", version = "0.15.0-dev" }
 bevy_ecs = { path = "../bevy_ecs", version = "0.15.0-dev" }
 bevy_gizmos = { path = "../bevy_gizmos", version = "0.15.0-dev", features = [
-  "bevy_render"
+  "bevy_render",
 ] }
 bevy_hierarchy = { path = "../bevy_hierarchy", version = "0.15.0-dev" }
 bevy_input = { path = "../bevy_input", version = "0.15.0-dev" }

